src/components/html.cjsx

```diff
@@ -48,7 +48,8 @@ module.exports = React.createClass
     hasMath = root.querySelector('math')
 
     # Return immediatly if no [data-math] or <math> elements are present
-    return unless window.MathJax and (_.any(nodes) or hasMath)
+    # TODO: If the MathJax Queue is not available then MathJax has not loaded yet. Add a load callback to enqueue.
```

Comment:
This may be solved by something like:

```coffee
unless window.MathJax.Hub?.Queue
  # TODO: Make sure this callback is not added more than once
  MathJax_AuthorInit = window.MathJax.AuthorInit
  window.MathJax.AuthorInit = => 
    MathJax_AuthorInit()
    if @isMounted()
      @setState({})
```